docs/features/wait/host_port.md

```diff
@@ -0,0 +1,31 @@
+# HostPort wait strategy
```

Comment:
```suggestion
# HostPort Wait strategy
```
Not saying we should write it like this, but it aligns it with the other strategies.